config/index.js

```diff
@@ -56,9 +56,10 @@ module.exports = {
   NO_CACHE: process.env.CACHE_ASSETS ? false : IS_DEV,
   FEEDBACK_URL: process.env.FEEDBACK_URL,
   SUPPORT_EMAIL: process.env.SUPPORT_EMAIL,
-  BUILD_DATE: process.env.APP_BUILD_DATE,
-  BUILD_BRANCH: process.env.APP_BUILD_TAG,
-  GIT_SHA: process.env.APP_GIT_COMMIT,
+  APP_BUILD_DATE: process.env.APP_BUILD_DATE,
+  APP_BUILD_BRANCH: process.env.APP_BUILD_BRANCH,
+  APP_BUILD_TAG: process.env.APP_BUILD_TAG,
+  APP_GIT_SHA: process.env.APP_GIT_COMMIT,
```

Comment:
I think this is bending the rules a bit.
According to the code above, I could change the GIT SHA, the build tag or the build date **at run time**.
I'm not sure if we practice immutable builds, but this going against it.
All of these properties are properties of the container and should be packaged (statically) with the app. You should not be able to change at runtime.
CC @mafonso

Reply:
This wasn't introduced in this PR - can we raise an issue for it?

Reply:
Yeah, I guess we could render them into a file from the ARGs into a file, and then consume that file, as a matter of principle. 
But if we agree not to override them in runtime, it has the same effect in a simpler way. And doing it does not have a tremendous side effect. It might actually be useful in test environments.

Reply:
https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/issues/692